Assets/_PlatformerDevelopment/Scripts/Debugger/PlayerInputDebugger.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Sirenix.OdinInspector;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace PersonalDevelopment
+{
+    public class PlayerInputDebugger : MonoBehaviour
+    {
+        private const string KeyboardAndMouseWASD = "Keyboard&Mouse_WASD";
+        private const string KeyboardAndMouseKeys = "Keyboard&Mouse_Keys";
+        private PlayerInput _playerOne = null;
+        private PlayerInput _playerTwo = null;
+        
+        private void Awake()
+        {
+            StorePlayerInputs();
+        }
+
+        private void StorePlayerInputs()
+        {
+            var allPlayers = FindObjectsOfType<PlayerInput>();
+            foreach (var player in allPlayers)
```

Comment:
do you want to make a check here to make sure that we only have, at max, 2 players just in case

Reply:
no, even if it's more than 2 players, there won't be any issues anyway.